benchmarks/dynamo/check_memory.py

```diff
@@ -0,0 +1,52 @@
+import argparse
+import sys
+import textwrap
+
+import pandas as pd
+
+
+def main(args):
+    actual = pd.read_csv(args.actual)
+    expected = pd.read_csv(args.expected)
+    failed = []
+
+    for name in actual["name"]:
+        actual_memory_compression = float(
+            actual.loc[actual["name"] == name]["compression_ratio"]
+        )
+        try:
+            expected_memory_compression = float(
+                expected.loc[expected["name"] == name]["compression_ratio"]
+            )
+        except TypeError:
+            print(f"{name:34} is missing from {args.expected}")
+            continue
+        if actual_memory_compression >= expected_memory_compression * 0.95:
```

Comment:
I was hoping 95% is enough for checking, but it seems pretty fragile, like the following failing after a rebase,
`dm_nfnet_f0                        actual_memory_compression=0.74, expected_memory_compression=0.79, FAIL`
(https://github.com/pytorch/pytorch/actions/runs/3515946230/jobs/5892272405)